lua/r/config.lua

```diff
@@ -504,6 +503,14 @@ local config = {
     tmpdir              = "",
     user_login          = "",
     user_maps_only      = false,
+    view_df = {
+        open_app = "",  -- How to open the CSV in Neovim or an external application.
+        how = "tabnew", -- How to display the data within Neovim if not using an external application.
+        csv_sep = "",   -- Field separator to be used when saving the CSV.
```

Comment:
```suggestion
        csv_sep = "",   -- Field separator to be used when saving the TSV.
```
Should standardize TSV/CSV everywhere.